src/components/DagreGraph/GraphRoot.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC, ReactNode, useEffect, useMemo, useRef } from 'react';
+import ResizeSensor from 'css-element-queries/src/ResizeSensor';
+
+import { useSetGraphWidth, useGraphCoordinates } from './GraphProvider';
+import useStyles from './GraphRoot.styles';
+
+interface GraphRootProps {
+  children: ReactNode | ReactNode[];
+}
+
+const GraphRoot: FC<GraphRootProps> = ({ children }) => {
+  const styles = useStyles();
+  const graphRef = useRef<HTMLDivElement>(null);
+  const setGraphWidth = useSetGraphWidth();
+  const { nodes: nodeCoordinates } = useGraphCoordinates();
+  const graphWidth = useMemo(() => {
+    return Math.ceil(
+      150 +
```

Comment:
Where is this 150 coming from? Is this just a padding?

Reply:
Yep, this is just padding so that the graph doesn't rest right up against the edge.

Reply:
I've moved the number into a `GRAPH_PADDING` variable to make it easier to see what it's used for